Formula/biointerchange.rb

```diff
@@ -0,0 +1,32 @@
+class Biointerchange < Formula
+  desc "Genomic linked data converter: an interchange GFF, GVF, VCF files via JSON"
+  homepage "https://github.com/BioInterchange/BioInterchangeC"
+  url "http://indie.kim/assets/biointerchange/biointerchange-2.0.5.tar.gz"
+  sha256 "36fcd19b39063fab87fb9f24a0752769ada91e81b82eb354ceb65d5147128141"
+  license "MIT"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
```

Comment:
Are you sure that `autoconf` and `automake` are needed? I built this formula from source myself on macOS 10.15 without `autoconf` installed.

Reply:
Good catch.